### PR TITLE
standardize README.md like for other repos

### PR DESCRIPTION
**Pull request by eysteins**
@pasinskim can you pls review, especially the "using the library" part?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,84 @@
...
+
+
+## Using the library
+
+To build the artifacts library, you can follow these steps:
+
+```
+git clone https://github.com/mendersoftware/artifacts.git
+cd artifacts
+go build
```
Comment (pasinskim) [inline README.md]: This is a library so you need to build using `go build ./...`.

File: README.md [hunk 2]
```diff
@@ -2,4 +2,84 @@
 [![codecov](https://codecov.io/gh/mendersoftware/artifacts/branch/master/graph/badge.svg)](https://codecov.io/gh/mendersoftware/artifacts)
 [![Go Report Card](https://goreportcard.com/badge/github.com/mendersoftware/artifacts)](https://goreportcard.com/report/github.com/mendersoftware/artifacts)
 
-# artifacts
+Mender Artifacts Library
```
Comment (maciejmrowiec) [inline README.md]: we should link to godoc it's a go package afterall

File: README.md [hunk 3]
```diff
@@ -2,4 +2,84 @@
...
+![Mender logo](https://mender.io/user/pages/04.resources/_logos/logoS.png)
+
+
+## Getting started
+
+To start using Mender, we recommend that you begin with the Getting started
+section in [the Mender documentation](https://docs.mender.io/).
+
+
+## Using the library
```
Comment (maciejmrowiec) [inline README.md]: it's a package use `go get`

subpackages should be exmaplained

File: README.md (hunk 1 above)
Comment (eysteins) [inline README.md]: ok, that didn't work for me (probably missing some PATH magic).
I remove this part as it seems to be pretty standard go stuff.

**Comment by eysteins** [comment]
@maciejmrowiec I can't explain the things you are asking. I will merge now as it is better than it was but please feel free to make it more crisp.

**Pull request merged**